app/actions/views/channel.js

```diff
@@ -333,6 +353,12 @@ export function refreshChannel(channelId) {
     };
 }
 
+export function refreshChannelWithRetry(channelId) {
+    return (dispatch, getState) => {
+        getPostsWithRetry(channelId)(dispatch, getState);
```

Comment:
Why aren't we marking the channel as refreshing any more? Doesn't this prevent the UI from being displayed for this?